src/Umbraco.Web.Common/Extensions/UmbracoCoreServiceCollectionExtensions.cs

```diff
@@ -293,6 +234,80 @@ public static IServiceCollection AddUmbracoCore(this IServiceCollection services
 
             coreRuntime.Configure(services);
 
+            // Add hosted services
+            services.AddHostedServices();
```

Comment:
I dont think this should be part of AddUmbracoCore. I guess it could be a separate extension on `IUmbracoBuilder`, like we have 
```
        public static IUmbracoBuilder WithCore(this IUmbracoBuilder builder)
            => builder.AddWith(nameof(WithCore), () => builder.Services.AddUmbracoCore(builder.WebHostEnvironment, builder.Config));
```

And then of cause called in`public static IUmbracoBuilder WithAllBackOfficeComponents(this IUmbracoBuilder builder)`

Reply:
Have updated to this.